### DISCUSSION TITLE
The JWT Token isn't getting removed after Expiry issue?

### DISCUSSION THREAD

**Discussion by sk1122** [HITK-TECH-Community/Community-Website#528] [General]
JWT Token has expired and then it is of no use, so deleting it after expiry is necessary?

Correct me if I am wrong?

@Kajol-Kumari

**Comment by Kajol-Kumari** [comment]
Hey @sk1122 The issue is that after the token expires we aren't doing any check for it in the frontend and so we are directing the user directly to the `admin` dashboard. That's the reason we want to clear the storage as soon as the token expires
